common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/wizards/TestProjectConfigurationWizardPage.xtend

```diff
@@ -47,42 +50,44 @@ class TestProjectConfigurationWizardPage extends WizardPage {
 	}
 
 	override void createControl(Composite superParent) {
-		val parent = new Composite(superParent, SWT.NONE)
-		parent.layout = new GridLayout(3, false)
+		val parent = new Composite(superParent, SWT.NONE) => [layout = new GridLayout(3, false)]
 		createBuildSystemSelectionArea(parent)
 		createFixtureSelectionArea(parent)
 		createDemoSelectionArea(parent)
 		setControl(buildSystem);
 	}
 
 	private def createDemoSelectionArea(Composite parent) {
-		demoCode = new Button(parent, SWT.CHECK)
-		demoCode.text = "Generate with examples"
+		demoCode = new Button(parent, SWT.CHECK) => [text = "Generate with examples"]
 	}
 
 	private def createFixtureSelectionArea(Composite parent) {
 		new Label(parent, SWT.NONE).text = "Available Fixtures:"
 		new Composite(parent, SWT.NONE)
 		new Label(parent, SWT.NONE).text = "Selected Fixtures:"
-		availableFixturesList = new List(parent, SWT.BORDER)
-		availableFixturesList.setData(Constants.SWT_BOT_ID_KEY, Constants.NEW_DIALOG_AVAILABLE_FIXTURE_LIST)
-		availableFixturesList.layoutData = new GridData(GridData.FILL_BOTH)
-		for (fixtureName : availableFixtureNames) {
-			availableFixturesList.add(fixtureName)
-		}
-		val buttonArea = new Composite(parent, SWT.NONE)
-		buttonArea.layout = new GridLayout(1, false)
-		val addBtn = new Button(buttonArea, SWT.BORDER)
-		addBtn.setData(Constants.SWT_BOT_ID_KEY, Constants.NEW_DIALOG_ADD_SELECTED_FIXTURE)
-		addBtn.text = ">"
-		addBtn.layoutData = new GridData(GridData.FILL_HORIZONTAL)
-		val delBtn = new Button(buttonArea, SWT.BORDER)
-		delBtn.text = "<"
-		delBtn.layoutData = new GridData(GridData.FILL_HORIZONTAL)
-		selectedFixturesList = new List(parent, SWT.BORDER)
-		selectedFixturesList.layoutData = new GridData(GridData.FILL_BOTH)
-		addBtn.addSelectionListener(createMoveListener(availableFixturesList, selectedFixturesList))
-		delBtn.addSelectionListener(createMoveListener(selectedFixturesList, availableFixturesList))
+		availableFixturesList = new List(parent, SWT.BORDER) => [
+			setData(Constants.SWT_BOT_ID_KEY, Constants.NEW_DIALOG_AVAILABLE_FIXTURE_LIST)
+			layoutData = new GridData(GridData.FILL_BOTH)
+		]
+		availableFixtureNames.forEach[availableFixturesList.add(it)]
+		val buttonArea = new Composite(parent, SWT.NONE) => [layout = new GridLayout(1, false)]
```

Comment:
Contents of the closure should be in a new line. But in general this is quite nice here as the initialization of the elements can be seen clearly.